### ISSUE TITLE
X & Y Pinout on Joystick Reversed

### ISSUE THREAD

**Issue by rdpowers** [CCHS-Melbourne/IoTuz#30]
Minor oversight in pinout of joystick. X pins are Y pins and Y pins are X pins.

**Comment by mage0r** [comment]
Are you sure this is still an issue?

At one point I switched the x and y over because the joystick was being mounted 90 degrees off on the board.

**Comment by rdpowers** [comment]
Yes. In reality, the X pot is aligned along the Y axis and vice-versa. Takes a second to realise that actuation means it needs to rotate about the tangent axis so intuition about what's X and what's Y is not normal.
  
![image](https://cloud.githubusercontent.com/assets/1632135/21667735/93d7b592-d34f-11e6-8db2-ab1543781952.png)

Also needs updating in documentation, so probably leave this open until that's fixed up as well.

**Comment by mage0r** [comment]
Hrm, I definitely re-labelled something to try and fix that.

It might have been the labels on the schematic.  but then, they're just text labels so they could have been fixed back up.

Is the pin list on the back wrong?

**Comment by rdpowers** [comment]
Yes, that is incorrect also in the same way.
